src/components/PlaybackControls/index.tsx

```diff
@@ -184,7 +167,8 @@ const PlayBackControls = ({
                 <p>
                     {roundedTime}{" "}
                     <span className={styles.lastFrameTime}>
-                        / {roundedLastFrameTime} {units[unitIndex]}
+                        / {roundedLastFrameTime}{" "}
+                        {timeUnits ? timeUnits.name : "s"}
```

Comment:
when is timeUnits falsy?

Reply:
When no file is loaded! And before TrajectoryFileInfo arrives?

Reply:
default initialize it to {1, "s"} ?  just to remove this conditional :)

Reply:
only if it makes sense.  no serious objection here.

Reply:
I think it's a UX question if we should have any time units when there is no data. It could be "-/-" or some other kind of placeholder

Reply:
Hm..... Okay, I'll just leave this here as is for now. This preserves the current behavior which is displaying "0 / 0 s" by default

Reply:
Actually, I cleaned this up in my latest commits, followed @meganrm 's [suggestion](https://github.com/allen-cell-animated/simularium-website/pull/115#discussion_r609004851) to take care of the unit label in the selector.